modules/ROOT/pages/bare-metal.adoc

```diff
@@ -113,3 +117,71 @@ IPAPPEND 2
 == PXE rootfs image
 
 include::pxe-artifacts.adoc[]
+
+Proposed addition to https://docs.fedoraproject.org/en-US/fedora-coreos/bare-metal/
+
+== Automated ISO installs with Ignition embedding
```

Comment:
This section equally applies to live PXE with `coreos-installer iso ignition embed` swapped for `coreos-installer pxe ignition wrap`.

Hmm, maybe we should restructure this page a bit. E.g. we keep this page about installing, and have a new page for live environments being useful on their own. And then here we have a section called e.g. "Installing from live ISO or PXE using Ignition" which provides the Ignition config, but otherwise links to the live page for more information about the live environments in general and the `embed`/`wrap` commands. WDYT? (We can do that bit in a follow-up!)

Reply:
OK, for now I just reworked the title and also noted the existence of `coreos-instalelr pxe ignition wrap` (thanks for pointing that out!)